### ISSUE TITLE
Duplication of Netherlands/Overijssel_2/2020

### ISSUE THREAD

**Issue by pvanheus** [nextstrain/ncov#174]
The virus name Netherlands/Overijssel_2/2020 is associated with both EPI_ISL_414550 and EPI_ISL_414460, with the former being dated later than the latter. I don't see an easy way to exclude one of what appear to be duplicates, but the duplicate name causes processing problems.

**Comment by trvrb** [comment]
Hi @pvanheus. We've just merged #59 that should fix this duplicates issue. After downloading from GISAID, run
```
./scripts/normalize_gisaid_fasta.sh data/gisaid_cov2020_sequences.fasta data/sequences.fasta
```
and then proceed with `snakemake` or `nextstrain build`, but let us know if this doesn't resolve things.

**Comment by pvanheus** [comment]
Thanks, I have my own Python [script](https://github.com/pvanheus/ncov/blob/master/combine_gisaid.py) that does the same as that script.
